### PR TITLE
Closes #51

### LINKED ISSUE #51: bug: when passing a combat outcome event to chatGPT don't mention damages
Otherwise we get a mention of the HP that were dealt: 

```
________________________
______Prompt______
Dialogue Start:
Nancy: "The war's toll was heavy on Hueteumuam. How can they be both victor and vanquished?"
James: "It's the strangest outcome. Could there have been foul play, or maybe a betrayal within their own ranks?"
Daniel: "Or the ground itself rebelled. Think of the noises we've heard, could they have been related to this?"
Lisa: "I wouldn't dismiss Daniel's warning so quickly. The earth tells its own tales, and we ought to listen."
Paul: "But 700 in damages... How will this affect our trade? They may have won, but at what cost?"
Nancy: "It could be a chance for Nikun Hihni. With Hueteumuam weakened, we might negotiate more favorable terms."
James: "I'll keep my ear to the ground during the feast. This war outcome might loosen tongues and reveal more than intended."
Lisa: "And I'll pay attention to any gestures from their ruler. It could tell us much about their true stance now."
Daniel: "Let's not forget, we need to stay alert. If the earth played a role in their war, it could disturb our own lands."
Paul: "Right, vigilance is key. But don't let fear stifle the feast. Strength is shown in unity and celebration, too."
________________________
```

### PR DESCRIPTION
**Pull request by cwastche**
Remove mentions of combat dmg in townhall discussion Also adds abritrary wait time when using mock

### CODE REVIEW

File: overlore/llm/constants.py [hunk 1]
```diff
@@ -45,7 +45,7 @@
...
+SYSTEM_STRING_TEMPLATE = """Imagine you're the game master for a strategy game, tasked with crafting dialogues for non-player characters (NPCs). Based on a brief character description provided in the format \"\"\"NPCs: <descriptions>\"\"\" and a key event described as \"\"\"event: <event>\"\"\", create engaging dialogues that reflect the NPCs' reactions and perspectives on the event. For combat related events, don't mention the number of damages directly. For reference, 10 is low, 100 is high in damages Ensure each NPC speaks no more than twice and maintain consistency with their character traits. {previous_summary_mention} Introduce a plot twist (such as a love interest, the death of a character, a natural disaster, drama, a magical beast appears) to add depth. Format each dialogue line as "<name>: <phrase>\n". Do not output anything else in the dialog. Conclude with "!end of discussion!". Afterwards, summarize the dialogue, outlining each NPC's contributions and reactions to the event. If unable to generate a conversation, reply with \"\"\"!failure:<enter reason of failure>!\"\"\".
```
Comment (edisontim) [inline overlore/llm/constants.py:48]: Missing `.` between `damages Ensure`

**Review by edisontim — CHANGES_REQUESTED** [review]

File: overlore/llm/constants.py (hunk 1 above)
Comment (edisontim) [reply overlore/llm/constants.py:48]: Are we sure 100 damages is high? Did you try to not give references?

Comment (cwastche) [reply overlore/llm/constants.py:48]: Numbers were kinda chosen randomly indeed. Not giving any references seem to work pretty well.

**Review by edisontim — APPROVED** [review]
lgtm

**Pull request merged**